specs/SHIELD.json

```diff
@@ -1021,6 +1035,110 @@
                 "example": "Privileged",
                 "title": "Type of security class the object(s) belongs to",
                 "type": "string"
+            },
+            "Shield.Deploy.Delta.Response": {
+                "title": "SHIELD - Deploy - Delta - Response",
+                "description": "Object returned as a result of query for cached data or re-evaluation operation.",
+                "type": "object",
+                "properties": {
+                    "invalid": {
+                        "additionalProperties": {
```

Comment:
The API doesn't respond with this set of data. Additional properties are only present on Kiota client parsing. The server sends the data normally.